backend/de.metas.device.scales/src/test/java/de/metas/device/scales/endpoint/TcpConnectionEndPointTest.java

```diff
@@ -129,8 +129,6 @@ public void run()
 								final String anotherWrongServerReturnString = MockedEndpoint.createWeightString(new BigDecimal(weight + 10)) + ISiscCmd.SICS_CMD_TERMINATOR;
 								out.write(anotherWrongServerReturnString.getBytes(ICmd.SICS_CMD_CHARSET));
 								System.out.println("TcpConnectionEndPointTest" + ": server-socked-thread replied with anotherWrongServerReturnString=" + anotherWrongServerReturnString);
-
-								out.flush();
```

Comment:
unrelated change that got in here in the context of #6205 .. need to check this myself